Added DishWasher Class and custom methods to file

mgm702 opened the pull request:
Created my own custom class out of a daily use object. The object that I decided to use was the Dish Washer. 

- The main purpose of the class is to load the dish washer and the soap.
- Conditionals set to check if both soap and dishes are loaded correctly.
- Also included methods to add/remove dishes from the dish washer.
- Made sure that Rubocop had no offenses. 
- Made it usable in irb and left comments on how to use the class.

[#105823864]

jasonnoble commented on objects/dish_washer.rb:
```diff
@@ -0,0 +1,56 @@
+    while continue_loading != 'no'
+      puts 'How many dishes would you like to load?'
+      @total_dishes += gets.chomp.to_i
+      puts 'Would you like to add more dishes?'
```
May want to let them know to type in `no` to exit.

jasonnoble commented on objects/dish_washer.rb:
```diff
@@ -0,0 +1,56 @@
+    if @soap == true && @total_dishes < 50
+      puts 'The dishwasher has succesfully started'
+      puts "You've successfully washed #{@total_dishes} dishes"
+      @total_dishes = 0
```
Don't you have to unload the dishwasher to get to 0 dishes?

mgm702 commented:
- I added a unload_dish_washer method and it's run after the dish washer has run through and cleaned the dishes.
- I also informed the user that if they wanted to exit the loop then they must enter 'no'.
- Refactored methods to correct Rubocop errors as well.
